cmd/cmd.go

```diff
@@ -72,6 +73,12 @@ func Go() {
 		// We only need to parse the message to get a recipient if none where
 		// provided on the command line.
 		recip = append(recip, msg.Header.Get("To"))
+		for _,cc := range strings.Split(msg.Header.Get("Cc"), ",") {
+			recip = append(recip, strings.TrimSpace(cc))
+		}
+		for _,bcc := range strings.Split(msg.Header.Get("Bcc"), ",") {
```

Comment:
I am not sure if this works correctly. As far as I know the Bcc is no part of the header, right?

Reply:
Worked for me, I was using my fork for a while.

Reply:
This probably won't work correctly if there's a comma as part of the recipient's name (in quotes).

Reply:
Didnt think of that use case, good catch. Unfortunately I no longer use msendmail, if someone else wants to pick up this PR please feel free. I don't think it will get merged however. I've been using swaks with maildev instead.